### DISCUSSION TITLE
Is there is reason WebAssembly is undefined on github.com?

### DISCUSSION THREAD

**Discussion by guest271314** [community/community#139462] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there is reason `WebAssembly` is undefined on github.com?

![Screenshot_2024-09-23_00-02-29](https://github.com/user-attachments/assets/ee138c4f-14fe-4883-97b5-329abb098498)

**Comment by fruetalo182** [comment]
Did you tried a different browser?

**Reply by guest271314** [reply]
`WebAssembly` *is* defined on this same page in Firefox Nightly 132.0a1 (2024-09-22) (64-bit).

**Reply by guest271314** [reply]
It's probably a Chromium flag.
